### PR TITLE
Checkout tags for local install

### PR DESCRIPTION
**Pull request by jbusecke**
Following @keewis suggestion in https://github.com/pangeo-forge/pangeo-forge-recipes/pull/595#issuecomment-1812104137 adding this to checkout tags

### CODE REVIEW

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -24,6 +24,8 @@ jobs:
         python-version: ["3.9", "3.10", "3.11"]
     steps:
       - uses: actions/checkout@v4
+        with:
+          depth: 0 # checkout tags (not done by default)
```
Comment (keewis) [inline .github/workflows/main.yaml:28]: apologies, the correct setting is `fetch-depth` not `depth`:
```suggestion
          fetch-depth: 0 # checkout tags (not done by default)
```

**Comment by jbusecke** [comment]
Ah thanks for catching this @keewis (sorry I should have double checked this myself, but am currently happily hacking away on other things). Also good to know you have Pangeo-Forge on your radar in general!

**Comment by keewis** [comment]
looks like CI didn't run for whatever reason?

**Comment by jbusecke** [comment]
Weird...Is this the correct [run](https://github.com/pangeo-forge/pangeo-forge-recipes/actions/runs/6881942031/job/18719379095)? Let me try to rerun this.

**Comment by jbusecke** [comment]
Seems like this is some gh issue. If you click on the small status symbol next to the commits they are displayed...
image

**Comment by keewis** [comment]
In any case, looks like the installed version is correct now.

**Comment by jbusecke** [comment]
Yep and the tests seem to show up again (at least on mobile). I think this is ready to merge.

**Pull request merged**